src/com/biocollapse/model/LiveStatistics.java

```diff
@@ -0,0 +1,60 @@
+package src.com.biocollapse.model;
+
+public class LiveStatistics {
+
+    private int infected;
+    private int recovered;
+    private int immune;
+    private int deaths;
```

Comment:
What i think is missing here is the current hospital capacity, which we wanted to display. The current plan in backend was to provide the percentage of all hospital capacities. So maybe we can display it like a progress bar or something? What do you think about it?

Reply:
Sounds cool. Depending on if we use the JFreeChart library it would also be interesting to visualise the deaths going up when hospital capacity is at its max.

Reply:
Thanks for the review!